packages/location-management/src/components/LocationTree/index.tsx

```diff
@@ -16,17 +16,18 @@ import { getHierarchyNodeFromArray } from '../../ducks/locationHierarchy/utils';
 import { SEARCH } from '../../lang';
 reducerRegistry.register(reducerName, reducer);
 
+reducerRegistry.register(reducerName, reducer);
```

Comment:
The reducer is registered twice?

Reply:
That probably happened during conflicts resolution, adressed via https://github.com/OpenSRP/web/pull/508/commits/3ae4fb8bd66f749fe5cb36196a34b7e60fe85702